template/utils/send_sample_request_utils.js

```diff
@@ -47,5 +47,9 @@ define([], function () {
         return result;
     }
 
-    return {handleNestedAndParsingFields};
+    function convertPathParams(url) {
```

Comment:
please add a little comment on top of the function to explain what it does and when it's used :)

Reply:
Like this?